packages/insomnia-app/app/common/migrate-from-designer.js

```diff
@@ -214,7 +214,7 @@ export default async function migrateFromDesigner({
       // For each workspace coming from Designer, mark workspace.scope as 'designer'
```

Comment:
Need to update the comment here as well
```suggestion
      // For each workspace coming from Designer, mark workspace.scope as 'design'
```